_rules/device-motion-user-interface-7677a9.md

```diff
@@ -26,19 +26,21 @@ htmlHintIgnore:
 
 ## Applicability
 
-The rule applies to an [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
+The rule applies to any [HTML document][] with an associated [Window object][] that has an [event listener list][] with one or more [event listeners][event listener] for [device orientation events][device orientation] or [device motion events][device motion].
 
 ## Expectation 1
 
-For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, a set of one or more [instruments][instrument] is available supporting the same outcomes of the event.
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, a set of one or more [instruments][instrument] is available in the same [web page][] supporting outcomes indistinguishable of those from the event.
```

Comment:
I don't think we can use the word "outcome" here. That's already defined in ACT and has a different meaning from how it is used here.

Reply:
Replaced "outcome" with one of its synonyms: "result"